webapp/app/templates/basis/base.html

```diff
@@ -29,35 +29,45 @@
 
     <script src="{{ static_url_for('static', filename='js/jquery-3.5.1.min.js') }}" type=text/javascript></script>
     {% if plausible_domain %}
-        <script async defer data-domain="{{ plausible_domain }}" src="https://plausible.io/js/plausible.js"></script>
+    <script async defer data-domain="{{ plausible_domain }}" src="https://plausible.io/js/plausible.js"></script>
     {% endif %}
-    <title>{{ _('page.title.error-prefix') if form and form.errors else '' }}
-        {{ header_title if header_title else (render_info.header_title if render_info and render_info.header_title else _('page.title')) }}</title>
+    <title>
+        {{ _('page.title.error-prefix') if form and form.errors else '' }}
+        {{ header_title if header_title else (render_info.header_title if render_info and render_info.header_title else _('page.title')) }}
+    </title>
 </head>
 
 <body>
-<div id="skip-to-content-div" class="col">
-    <a href="#main" id="skip-to-content-link">{{ _('skip-to-content-link') }}</a>
-</div>
-<div class="container-fluid p-0">
-    {% include 'basis/mobile_nav.html' %}
-    {% include 'basis/sidebar.html' %}
-    <main id="main" class="main">
-        <div class="container-fluid p-0 d-flex flex-column min-vh-100">
-            {% block flashed_messages %}
+    <div id="skip-to-content-div" class="col">
+        <a href="#main" id="skip-to-content-link">{{ _('skip-to-content-link') }}</a>
+    </div>
+    <div class="container-fluid p-0">
+        {% include 'basis/mobile_nav.html' %}
+        {% include 'basis/sidebar.html' %}
+        <main id="main" class="main">
+            <noscript>
+                {% include 'error/js_disabled.html' %}
+
+                <!-- If javascript is disabled, the main_content container is hidden. -->
+                <style type="text/css">
+                        #main_content { display: none!important; }
+                </style>
```

Comment:
I don't think we ever use <style> tags so far. I just think it's interesting. Did you get the idea from somewhere? (i.e., is this the best practice way of handling disabled JavaScript that I should memorize? :D )

Reply:
Yes, I think with a slow computer the web page might flicker. Others also have <meta> in the <noScript> container to redirect to another page. But this is probably not supported by each browser.